resources/helm/overlays/istio_cni/templates/daemonset.yaml

```diff
@@ -56,51 +56,6 @@ spec:
       containers:
         # This container installs the Istio CNI binaries
         # and CNI network config file on each node.
-{{- if and (.Values.cni.supportedReleases) (has "v1.1" .Values.cni.supportedReleases) }}
```

Comment:
Should we keep the v1.1 cni container in the template ?

Reply:
I think you removed the v1.1 images, right?

Reply:
right, I am think what will happen in the existing cluster. If a set of v1.1 images had been there and they will get operator upgrade, will those v1.1 containers be deleted ?